wasm2c/examples/fac/main.c

```diff
@@ -5,7 +5,8 @@
 
 int main(int argc, char** argv) {
   /* Make sure there is at least one command-line argument. */
-  if (argc < 2) return 1;
+  if (argc < 2)
```

Comment:
I think he wabt (aka Chromium) style is to always just curly braces if you put the body on its own line

Reply:
This was a clang-format autochange; restored to original.